### PR TITLE
Updating readme, bootstrap now part of main script

### PR DESCRIPTION
**Pull request by pookey**
Bootstrapping is now part of the main script, updating README to relect this.  
A few minor formatting tweaks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,16 +6,16 @@ A framework for controlling multi-environment multi-component terraform-managed
 
 Terraform scaffold consists of a terraform wrapper bash script, a bootstrap script and a set of directories providing the locations to store terraform code and variables.
 
-| Thing	| Things about the Thing |
-|-------|------------------------|
+| Thing	           | Things about the Thing |
```
Comment (Zordrak) [inline README.md]: Is this spacing change necessary?

Comment (pookey) [reply README.md]: Not required - however it does make it much easier to read if it's not been Markedup.  Just lines it up a bit.

Comment (Zordrak) [reply README.md]: However also means:
* all subsequent modifications need to take care of the alignment. An additional line of longer length means a commit modifying every line in the table.
* Anyone choosing to edit in a variable-width font will struggle to determine the correct alignment.

Markdown's ignorance of the spaces in table formatting is a valuable feature.

If you can pull these changes, the others are fine for merge.

Comment (pookey) [reply README.md]: Ok, makes sense.    I'll close this request for now as I've seen another small issue I'll bundle in at a later date.  Thanks!

**Pull request closed without merging**